platform/commonUI/edit/src/actions/SaveAction.js

```diff
@@ -127,17 +118,13 @@ define(
             function doSave() {
                 //This is a new 'virtual object' that has not been persisted
                 // yet.
-                if (!domainObject.getModel().persisted){
+                if (domainObject.getModel().persisted === undefined){
                     return getParent(domainObject)
-                            .then(doWizardSave)
-                            .then(persistObject)
-                            .then(getParent)//Parent may have changed based
-                                            // on user selection
-                            .then(locateObjectInParent)
-                            .then(function(){
-                                return fetchObject(domainObject.getId());
-                            })
-                        .catch(doNothing);
+                        .then(doWizardSave)
+                        .then(getParent)
+                        .then(copyService.perform.bind(copyService, domainObject))
```

Comment:
Ah, good callout. Yes, as a consequence of this any links get converted to new originals. Which is not what we want (e.g. in basic use case of Create Display Layout, drag stuff in, save).

In this case, I think a semi-deep copy (as current copy, except don't copy links) is sufficient. This wouldn't address cases where a Move is done during edited (Move Object A into new Layout being Edited, Save, results in any existing links to Object A being broken). But we disallow Move during editing, so this shouldn't be reproducible in practice.